x-pack/plugins/apm/server/lib/helpers/convert_ui_filters/get_ui_filters_es.ts

```diff
@@ -44,14 +38,11 @@ export function getUiFiltersES(
   return esFilters;
 }
 
-function getKueryUiFilterES(
-  indexPattern: IIndexPattern | undefined,
-  kuery?: string
-) {
-  if (!kuery || !indexPattern) {
+function getKueryUiFilterES(kuery?: string) {
+  if (!kuery) {
     return;
   }
 
   const ast = esKuery.fromKueryExpression(kuery);
-  return esKuery.toElasticsearchQuery(ast, indexPattern) as ESFilter;
+  return esKuery.toElasticsearchQuery(ast) as ESFilter;
```

Comment:
Was this not needed?

Reply:
I tested it by entering various searches in the query bar, and using the contextual filters.

Reply:
It says:
```
 * IndexPattern isn't required, but if you pass one in, we can be more intelligent
 * about how we craft the queries (e.g. scripted fields)
```

Sounds like it should be okay for our use case?

Reply:
yeah, I think I've seen that before and wondered what that actually meant :D
If it's just scripted fields and you did the manual testing it's probably fine